.github/workflows/repository-management.yml

```
@@ -136,10 +136,14 @@ jobs:
           echo "new_version=$NEW_VERSION" >> "$GITHUB_OUTPUT"
 
       - name: Update provider version in version.go
+        with:
```

```suggestion
        env:
```

⛏️  Should we use an `env` block here in order to stay consistent with the step definitions before?